pymc_marketing/clv/plotting.py

```diff
@@ -344,15 +346,138 @@ def plot_probability_alive_matrix(
+    incremental.
+
+    Adapted from legacy ``lifetimes`` library:
+    https://github.com/CamDavidsonPilon/lifetimes/blob/master/lifetimes/plotting.py#L392
+
+    Parameters
+    ----------
+    model :
+        A fitted ``BetaGeoModel`` or ``ParetoNBDModel``.
+    purchase_history : ~pandas.DataFrame
+        A Pandas DataFrame containing *customer_id_col* and *datetime_col*.
+    customer_id_col : string
+        Column in the *transactions* DataFrame denoting the *customer_id*.
+    datetime_col :  string
+        Column in the *transactions* DataFrame denoting datetimes purchase were made.
+    t : int
+        Number of time units since earliest purchase to include in plot.
+    plot_cumulative : bool
+        Default: *True*
+        Plot cumulative purchases over time. Set to *False* to plot incremental purchases.
+    t_cal : int, optional
+        If testing model on unobserved data, specify number of time units in training data to add an indicator for
+        the start of the testing period.
+    datetime_format : string, optional
+        A string that represents the timestamp format. Useful if Pandas doesn't recognize the provided format.
+    time_unit : string, optional
+        Time granularity for study.
+        Default: 'D' for days. Possible values listed here:
+        https://numpy.org/devdocs/reference/arrays.datetime.html#datetime-units
+    time_scaler : int, optional
+        Default: 1. Scales *recency* & *T* to a different time granularity.
+        This is useful for datasets spanning many years, and running predictions in different time scales.
+    sort_transactions : bool, optional
+        Default: *True*
+        If *transactions* DataFrame is already sorted in chronological order, set to *False* to improve computational
+        efficiency.
+    set_index_date : bool, optional
+        Set to True to return a dataframe with a datetime index.
+    title : str, optional
+        Figure title
+    xlabel : str, optional
+        Figure xlabel
+    ylabel : str, optional
+        Figure ylabel
+    ax : matplotlib.AxesSubplot, optional
+        Using user axes
+    kwargs
+        Additional arguments to pass into the pandas.DataFrame.plot command.
+
+    Returns
+    -------
+    axes: matplotlib.AxesSubplot
+
+    References
+    ----------
+    .. [1] Fader, Peter S., Bruce G.S. Hardie, and Ka Lok Lee (2005),
+    A Note on Implementing the Pareto/NBD Model in MATLAB.
+    http://brucehardie.com/notes/008/
+    """
+    # clear any existing pyplot figures
+    plt.clf()
+
+    if ax is None:
+        ax = plt.subplot(111)
+
+    df_cum_transactions = _expected_cumulative_transactions(
+        model,
+        purchase_history,
+        customer_id_col,
+        datetime_col,
+        t,
+        datetime_format,
+        time_unit,
+        time_scaler,
+        sort_transactions,
+        set_index_date,
+    )
```

Comment:
This function has a lot of arguments. Shall we use keyword arguments to make sure we do not pass the wrong order

```python
    df_cum_transactions = _expected_cumulative_transactions(
        model=model,
        ...
    )
```

Reply:
+1